src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/SortQuery.java

```diff
@@ -43,6 +63,21 @@ private static List<Sort> generateSortList(List<SortingParameter> sortingParamet
         }).filter(Objects::nonNull).collect(Collectors.toList());
     }
 
+    private static List<Order> generateOrders(List<SortingParameter> sortingParameters,
+                                              CriteriaBuilder builder,
+                                              Root<TaskResource> root) {
+        return sortingParameters.stream().map((sortingParameter) -> {
+            switch (sortingParameter.getSortOrder()) {
```

Comment:
New line following re-engineering for performance. Can we add unit test to cover this line?